cli/src/main/java/com/devonfw/tools/ide/io/FileAccessImpl.java

```diff
@@ -674,18 +674,15 @@ public static String generatePermissionString(int permissions) {
     return permissionStringBuilder.toString();
   }
 
-  private void extractArchive(Path file, Path targetDir, Function<InputStream, ArchiveInputStream> unpacker) {
+  private void extractTarArchive(Path file, Path targetDir, Function<InputStream, ArchiveInputStream> unpacker) {
```

Comment:
Would it be possible to use `TarArchiveInputStream` instead of `ArchiveInputStream` since we use this method just for tar archives? Is there a benefit to avoid castiing?

Reply:
I understand. Didn't know about the return type `getNextEntry`. Thank you for clarification.

Reply:
Can be resolved